api/swagger.yml

```diff
@@ -610,6 +610,8 @@ definitions:
       currentStep:
         type: integer
         format: int64
+      releasePage:
```

Comment:
What do you think about calling this something like `html_url`?

We already know it's for a release because it belongs to a release object.

Reply:
done